### DISCUSSION TITLE
PLEASE HELP : ERROR IN BYTECODE // URGENT

### DISCUSSION THREAD

**Discussion by RaghavShah** [smartcontractkit/full-blockchain-solidity-course-py#1719] [Q&A] [answered]
![error](https://user-images.githubusercontent.com/76104964/192331825-4e2d5043-f39d-445a-a7db-917f63bca9cf.png)

please can anyone resolve it I'm stuck here and I'm unable to move ahead

**Comment by cromewar** [comment, marked as answer]
Hello @RaghavShah 
You have a missing closing statement on line 37, you should close "abi" with another ]
